CMakeLists.txt

```diff
@@ -3,29 +3,22 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 
 project(meta-json-parser VERSION 0.1 LANGUAGES CXX CUDA)
 
-find_package(CUDAToolkit 10.0 REQUIRED)
+# As of now not using LIBCUDF throws
+set(USE_LIBCUDF 1)
 
-IF(NOT LOCAL_LIB)
-find_package(Boost 1.75)  # version 1.75 is required to use boost::mp11::mp_pairwise_fold_q
-ENDIF()
+find_package(CUDAToolkit EXACT 11.8 REQUIRED)
```

Comment:
currently the newest is 12 (so the previous would find 12, as it only set minimum), but RAPIDS are currently limited to 11.X